sunkit_instruments/goes_xrs/goes_chianti_tem.py

```diff
@@ -0,0 +1,212 @@
+from astropy import units as u 
+from astropy.io import fits
+from astropy.time import Time
+from sunpy import timeseries as ts 
+from sunpy.time import parse_time
+from sunpy.data import manager
+from scipy import interpolate
+import numpy as np 
+import pandas as pd
+from sunpy.util.exceptions import warn_user
+
+__all__ = ["goes_calculate_temperature_em", "_goes_chianti_temp_em", "_manage_goesr_detectors"]
+
+
+def goes_calculate_temperature_em(goes_ts, abundance="coronal", remove_scaling=False):
```

Comment:
```suggestion
def calculate_temperature_emiss(goes_ts, abundance="coronal", remove_scaling=False):
```

I don't think we need goes in the function name.